aws/logs_monitoring/README.md

```diff
@@ -22,6 +27,83 @@ AWS Lambda function to ship logs from S3 and CloudWatch, custom metrics and trac
 1. Set up triggers to the installed Forwarder either [automatically](https://docs.datadoghq.com/integrations/amazon_web_services/?tab=allpermissions#automatically-setup-triggers) or [manually](https://docs.datadoghq.com/integrations/amazon_web_services/?tab=allpermissions#manually-setup-triggers).
 1. Repeat the above steps in another region if you operate in multiple AWS regions.
 
+<details><summary>CloudFormation Parameters</summary>
+
+Required
+- `DdApiKey` - Your Datadog API Key. This can be found in DataDog, under Integrations > APIs > API Keys.
+    The API Key will be stored in AWS Secrets Manager.
+- `DdSite` - The Datadog site that your metrics and logs will be sent to. Should either be `datadoghq.com` 
+    or `datadoghq.eu`
+
+Lambda Function (Optional)
+- `FunctionName` - The name of the DataDog Forwarder Lambda function. DO NOT change when updating an 
+    existing CloudFormation stack, otherwise the current forwarder function will be replaced and all the 
+    triggers will be lost.
+- `MemorySize` - Memory size for the Datadog Forwarder Lambda function
+- `Timeout` - Timeout for the Datadog Forwarder Lambda function
+- `ReservedConcurrency` - Reserved concurrency for the Datadog Forwarder Lambda function
+- `LogRetentionInDays` - CloudWatch log retention for logs generated by the Datadog Forwarder Lambda 
+    function
+
+
+Log Forwarding (Optional)
+- `DdTags` - Add custom tags to forwarded logs, comma-delimited string, no trailing comma, e.g., 
+    `env:prod,stack:classic`
+- `DdMultilineLogRegexPattern` - Use the supplied regular expression to detect for a new log line for 
+    multiline logs from S3, e.g., use expression `\d{2}\/\d{2}\/\d{4}` for multiline logs beginning 
+    with pattern "11/10/2014".
+- `DdUseTcp` - By default, the forwarder sends logs using HTTPS through the port 443. To send logs over a 
+    SSL encrypted TCP connection, set this parameter to true.
+- `DdNoSsl` - Disable SSL when forwarding logs, set to true when forwarding logs through a proxy.
+- `DdUrl` - The endpoint URL to forward the logs to, useful for forwarding logs through a proxy
+- `DdPort` - The endpoint port to forward the logs to, useful for forwarding logs through a proxy
+- `DdSkipSslValidation` - Send logs over HTTPS, while NOT validating the certificate provided by the 
+    endpoint. This will still encrypt the traffic between the forwarder and the log intake endpoint, 
+    but will not verify if the destination SSL certificate is valid.
+- `DdUseCompression` - Set to false to disable log compression. Only valid when sending logs over HTTP.
+- `DdCompressionLevel` - Set the compression level from 0 (no compression) to 9 (best compression).
+- `DdForwardLog` - Set to false to disable log forwarding, while keep the forwarder forward other 
```

Comment:
`while keep the forwarder forward other observability data` -> `while continuing to forward other observability data`